coldfront/core/utils/mou.py

```diff
@@ -0,0 +1,182 @@
+from coldfront.core.utils.common import import_from_settings
+from coldfront.core.resource.utils_.allowance_utils.computing_allowance import ComputingAllowance
+import datetime
+
+def upload_to_func(instance, filename):
+    fs = import_from_settings('FILE_STORAGE')
+    path = ''
+    type_ = ''
+    from coldfront.core.allocation.models import (AllocationAdditionRequest,
+                                                  SecureDirRequest,
+                                                  AllocationPeriod)
+    from coldfront.core.project.models import SavioProjectAllocationRequest
+    if isinstance(instance, SavioProjectAllocationRequest):
+        path += fs['details']['NEW_PROJECT_REQUEST_MOU']['location']
+        type_ = 'NewProject'
```

Comment:
Does it make sense to place this type in the `FILE_STORAGE` setting (with a more descriptive name than "type")?

Reply:
what name would you recommend?